src/CBot/CBotStack.h

```diff
@@ -94,14 +94,22 @@ class CBotStack
      * \param[out] end Ending position in code of the error
      * \return Error number
      */
-    CBotError GetError(int& start, int& end) { start = m_start; end = m_end; return m_error; }
+    CBotError GetError(std::size_t& start, std::size_t& end) const
+    {
+        start = m_start;
+        end = m_end;
+        return m_error;
+    }
 
     /**
      * \brief Get last error
      * \return Error number
-     * \see GetError(int&, int&) for error position in code
+     * \see GetError(std::size_t&, std::size_t&) for error position in code
      */
-    CBotError GetError() { return m_error; }
+    CBotError GetError() const
```

Comment:
You could've moved this into CPP file while you were here but I think it was done by someone already in the `dev` branch.

I appreciate the effort to add `const` to methods though.